lib/cli/placeorder.ts

```diff
@@ -1,20 +1,21 @@
 import { Arguments, Argv } from 'yargs';
 import { Order, OrderSide, PlaceOrderEvent, PlaceOrderRequest, PlaceOrderResponse, SwapFailure, SwapSuccess } from '../proto/xudrpc_pb';
 import { callback, loadXudClient } from './command';
-import { coinsToSats, satsToCoinsStr } from './utils';
+import { checkDecimals, coinsToSats, satsToCoinsStr } from './utils';
 
 export const placeOrderBuilder = (argv: Argv, side: OrderSide) => {
   const command = side === OrderSide.BUY ? 'buy' : 'sell';
   argv.option('quantity', {
     type: 'number',
+    describe: 'the quantity for limit/market orders, less than 13 decimals',
   })
   .option('pair_id', {
     type: 'string',
     describe: 'the trading pair ticker id for the order',
   })
   .option('price', {
     type: 'string',
-    describe: 'the price for limit orders, or "mkt"/"market" for market orders',
+    describe: 'the price for limit orders, or "mkt"/"market" for market orders, less than 13 decimals',
```

Comment:
I personally think "max 12 decimal places" is more clear - although I also think this description can also be left alone as in most cases people aren't going to be trying to enter prices with that many decimal places anyway. Maybe better not to confuse anyone, if somebody tries a really weird price then they will get a more descriptive error message.